weatherman/utils.py

```diff
@@ -0,0 +1,44 @@
+import re as regex
+from typing import Tuple
+import os
+
+
+def extract_year_month(date: str) -> Tuple[int, int]:
+    year, month, *_ = map(int, date.split('-'))
+
+    return year, month
+
+
+def extract_year(date: str) -> int:
+    year = int(date.split('-')[0])
+
+    return year
+
+
+def extract_month(date: str):
+    month = int(date.split('-')[1])
+
+    return month
+
+
+def remove_whitespaces_from_start_end(text: str) -> str:
```

Comment:
Unused, remove all unused functions

Reply:
it has usage in other modules of application as specified in PR description